tests/parsers/appbio_quantstudio_designandanalysis/testdata/appbio_quantstudio_designandanalysis_QS7Pro_Genotyping_example07.json

```diff
@@ -349,11 +350,13 @@
                                             37,
                                             38,
                                             39,
-                                            40
+                                            40,
+                                            1
```

Comment:
This result seems unlikely to be desirable. Are we sure there isn't some other filter we should be applying _instead_ of stage number? 

E.g. the description mentions cycle number and reporter dye setting, perhaps one of those?

Reply:
Yes, the result is weird. However James and Joe told me this is the correct change.

Reply:
Thank you @alejandro-salgado

@james-leinas will follow your lead on approving

Reply:
@nathan-stender, @alejandro-salgado and I discussed async about handling stage number as filter for cases when it is present in the example file. Other cases we identified will not contain stage number and we will record all the values in the datacube for cycle number/reporter dye setting pair. This at least will help to filter when we have a basis to do so, otherwise we take the information as given to us in the file.

Reply:
Sorry not clear if this means there will be changes or if this is ready to do?

Reply:
Sorry for confusion, this will mean some more changes.